src/components/CategoricalField.tsx

```diff
@@ -12,7 +12,10 @@ function CategoricalField({
   return (
     <Autocomplete
       data-cy={`${label}-categorical-field`}
-      options={options.sort((a, b) => a.label.localeCompare(b.label))}
+      options={options.sort((a, b) => {
+        if (a.label === 'All') return -1;
```

Comment:
Here too: "All" introduces a bit of extra complexity without being very necessary. 
I think we could get rid of "All" and just use `Null`/unselected internally to mean the same.

Unless of course you added "All" because what I suggest is much harder. Then I'd add a comment to explain that.

Reply:
I believe we discussed and decided to explicitly add `All` for pipeline versions during stand up.

Reply:
To clarify this bit of code was added as a result of including `All` in the pipeline versions and the need to have condition to order `All` at the top of the dropdown options.